### PR TITLE
add instructions for AWS account

### PR DESCRIPTION
**Pull request by davejrt**

### CODE REVIEW

**Comment by nicksnyder** [comment]
Can you also add docs on how a distribution team member creates a new account with correct permissions?

**Comment by daxmc99** [comment]
Should also request input from the security team here as well to ask for guidance on baseline permissions for team members cc @sourcegraph/security

**Comment by nicksnyder** [comment]
Open a PR that documents what we do today, the request review

File: handbook/engineering/distribution/internal_infrastructure.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ Credentials: [1Password](https://my.1password.com/vaults/dnrhbauihkhjs5ag6vszsme
...
 We maintain separate AWS accounts with Sourcegraph instances and other infrastructure that mimic various customers' environments for testing purposes. See "[Customer environment replicas and managed instances](https://my.1password.com/vaults/dnrhbauihkhjs5ag6vszsme45a/003/ctqvj7zcmdiujmfh2mxzffdlym)" on 1Password for the list.
 
+### Accessing sourcegraph internal infrastrcture
+
+To access this AWS account:
+
+1. Request an AWS user ID by asking a member of the `#distribution` team in slack. 
+1. Once you have your credentials visit https://aws.amazon.com/console/.
+1. Sign into AWS using those details and Account ID `sourcegraph. 
+
```
Comment (unknwon) [inline handbook/engineering/distribution/internal_infrastructure.md:48]:
```diff
-1. Sign into AWS using those details and Account ID `sourcegraph.
+1. Sign into AWS using those details and Account ID `sourcegraph`.
```

**Pull request merged**